dev-python/pymacaroons/pymacaroons-0.13.0-r1.ebuild

```
@@ -0,0 +1,51 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1
+
+DESCRIPTION="PyMacaroons is a Python implementation of Macaroons."
```

```suggestion
DESCRIPTION="A Python implementation of Macaroons"
```

Don't repeat the package name and then it's no longer a full sentence, so no full stop.